docs/diagrams/ArchitectureDiagram.puml

```diff
@@ -5,29 +5,37 @@
 !include style.puml
 
 Package " "<<Rectangle>>{
-    Class UI UI_COLOR
-    Class Logic LOGIC_COLOR
-    Class Storage STORAGE_COLOR
-    Class Model MODEL_COLOR
-    Class Main #grey
-    Class Commons LOGIC_COLOR_T2
+    Class UIManager UI_COLOR
+    Class LogicManager LOGIC_COLOR
+    Class StorageManager STORAGE_COLOR
+    Class ModelManager MODEL_COLOR
+    Class AddressBookParser LOGIC_COLOR_T2
+    Class MainApp #grey
+    Class MainWindow #grey
+    Class Patients MODEL_COLOR_T1
```

Comment:
@tankh99 currently we havent refactor the name yet right?

Reply:
@vnnamng I was thinking this is okay because we want to eventually refactor to Patients anyway. And the diagrams can reflect the end result instead of what we have now